cmd/attach/policy/cmd.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package policy
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/aws/tags"
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+const (
+	use     = "policy"
+	short   = "Attach policy to a role"
+	long    = "Attach existing policy to role in the user’s AWS account"
+	example = `  # Attach policy <policy_arn> to role <role_name>
+  rosa attach policy --policy-arn=<policy_arn> --role-arn=<role_name>`
+)
+
+var args struct {
+	policyArn string
+	roleName  string
+}
+
+func NewAttachPolicyCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     use,
+		Short:   short,
+		Long:    long,
+		Example: example,
+		Args:    cobra.NoArgs,
+		Run:     rosa.DefaultRunner(rosa.RuntimeWithOCMAndAWS(), AttachPolicyRunner()),
+	}
+
+	flags := cmd.Flags()
+
+	flags.StringVarP(
+		&args.policyArn,
+		"policy-arn",
+		"p",
+		"",
+		"Policy arn of the policy to be attached to the specified role (required).",
+	)
+	flags.StringVarP(
+		&args.roleName,
+		"role-name",
+		"r",
+		"",
+		"Role name of the role to attach the specified policy (required).",
+	)
+	cmd.MarkFlagRequired("policy-arn")
+	cmd.MarkFlagRequired("role-name")
+
+	interactive.AddModeFlag(cmd)
+	return cmd
+}
+
+func AttachPolicyRunner() rosa.CommandRunner {
+	return func(_ context.Context, r *rosa.Runtime, cmd *cobra.Command, _ []string) error {
+		mode, err := interactive.GetMode()
+		if err != nil {
+			return err
+		}
+
+		defer r.Cleanup()
```

Comment:
@marcolan018 You don't need to call this, it's taken care of for you by the `DefaultRunner` 👍